app.py

```diff
@@ -79,6 +98,44 @@ def aboutus():
     return render_template("aboutus.html")
 
 
+@app.route("/submit-template", methods=["GET", "POST"])
+def submit_template():
+    form = TemplateSubmissionForm()
+    
+    if request.method == "POST" and form.validate_on_submit():
+        name = form.name.data
+        email = form.email.data
+        target_name = form.target_name.data
+        target_email = form.target_email.data
+        target_subject = form.target_subject.data
+        target_body = form.target_body.data
+
+        # Do something with the inputs
+        # createTemplate(...) --> emails.py:createTemplate(...)
+        d = {
+        'name' : name,
+        'email' : email,
+        'target_name' : target_name,
```

Comment:
Can we ask for topic submission too? then allow admins to adjust before inclusion into db